debugging/debugging.md

```diff
@@ -36,10 +36,18 @@
 
 ### Lesson
 
-[Debugging (video walkthrough of slides)](https://drive.google.com/file/d/1VMyGDG5CnNMkjCyaNx1LAjxTMKxC0QQ4/view?usp=sharing)
-
-[Debugging (slides)](https://docs.google.com/presentation/d/1Ol1as_RuxBpXMd4VxCUyvFVkjpcqAxg6B-c0Sl9KDLc/edit?usp=sharing)
-
+- Video walkthrough of slides [Debugging (12 mins watch)](https://drive.google.com/file/d/1VMyGDG5CnNMkjCyaNx1LAjxTMKxC0QQ4/view?usp=sharing)
+
+- Read through lesson slides [Debugging](https://docs.google.com/presentation/d/1Ol1as_RuxBpXMd4VxCUyvFVkjpcqAxg6B-c0Sl9KDLc/edit?usp=sharing)
+
+##### General Tips
```

Comment:
@vsharma20v , I think a good title would be like:
General Tips for Debugging(with VS Code)